### DISCUSSION TITLE
Code review IK-91

### DISCUSSION THREAD

**Discussion by Artur24327** [HowProgrammingWorks/LiveQA#49] [Code review]
https://github.com/Artur24327/TradeHelperBot.git

**Comment by Artur24327** [comment]
Написав бота, який парсить дані з біржі і дає потрібну інформацію трейдеру по запросу. Оскільки тільки вчусь, місцями код може бути жахливим, але я працюю над цим.

**Comment by tshemsedinov** [comment]
First review https://github.com/Artur24327/TradeHelperBot/issues/1
